salt/elife-bot/email-templates.sls

```diff
@@ -0,0 +1,9 @@
+#
+# email templates
+#
+
+install-email-templates:
+    git.latest:
```

Comment:
you might want to crib from here if you have problems:
https://github.com/elifesciences/elife-bot-formula/blob/master/salt/elife-bot/init.sls#L13-L21

I'd also recommend adding the force_* params to keep that repo free from accidental modifications.